src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -251,16 +253,22 @@ String populateTableSql() {
             for (int colNum = 0; colNum < totalColumns; colNum++) {
 
                 // TODO: add betterway to enclose data
-                if (JDBCType.CHAR == getColumn(colNum).getSqlType().getJdbctype()
+                if (JDBCType.CHAR == getColumn(colNum).getSqlType().getJdbctype() 
                         || JDBCType.VARCHAR == getColumn(colNum).getSqlType().getJdbctype()
                         || JDBCType.NCHAR == getColumn(colNum).getSqlType().getJdbctype()
                         || JDBCType.NVARCHAR == getColumn(colNum).getSqlType().getJdbctype()
                         || JDBCType.TIMESTAMP == getColumn(colNum).getSqlType().getJdbctype()
                         || JDBCType.DATE == getColumn(colNum).getSqlType().getJdbctype()
-                        || JDBCType.TIME == getColumn(colNum).getSqlType().getJdbctype())
+                        || JDBCType.TIME == getColumn(colNum).getSqlType().getJdbctype()) {
```

Comment:
**SUGGESTION**: In order to reduce complexity it is better to create new method having all conditions and call that method in your if condition. 
If method is reusable then you can move this method to Util class. Otherwise create this helper method in same class.